Pipfile

```diff
@@ -16,4 +16,4 @@ pip-tools = "*"
 
 [requires]
 
-python_version = "3.4"
+python_version = "3.5"
```

Comment:
can we bump further to 3.6 or 3.7?

Reply:
Will give this a try also. I recall seeing 3.6 used a lot out there.